BUILD

```diff
@@ -6,16 +6,16 @@ container_bundle(
     name = "all",
     images = {
         "gcr.io/{PROJECT_ID}/base:debug": "//base:debug",
-        "gcr.io/{PROJECT_ID}/base:debug-debian10": "//base:debug-debian10",
+        "gcr.io/{PROJECT_ID}/base:debug-debian10": "//base:debug_debian10",
         "gcr.io/{PROJECT_ID}/base:debug-nonroot": "//base:debug-nonroot",
-        "gcr.io/{PROJECT_ID}/base:debug-nonroot-debian10": "//base:debug-nonroot-debian10",
+        "gcr.io/{PROJECT_ID}/base:debug-nonroot-debian10": "//base:debug-nonroot_debian10",
         "gcr.io/{PROJECT_ID}/base:latest": "//base:base",
         "gcr.io/{PROJECT_ID}/base:nonroot": "//base:base-nonroot",
-        "gcr.io/{PROJECT_ID}/base:nonroot-debian10": "//base:base-nonroot-debian10",
-        "gcr.io/{PROJECT_ID}/static:debian10": "//base:static-debian10",
+        "gcr.io/{PROJECT_ID}/base:nonroot-debian10": "//base:base-nonroot_debian10",
+        "gcr.io/{PROJECT_ID}/static:debian10": "//base:static_debian10",
         "gcr.io/{PROJECT_ID}/static:latest": "//base:static",
         "gcr.io/{PROJECT_ID}/static:nonroot": "//base:static-nonroot",
-        "gcr.io/{PROJECT_ID}/static:nonroot-debian10": "//base:static-nonroot-debian10",
+        "gcr.io/{PROJECT_ID}/static:nonroot_debian10": "//base:static-nonroot_debian10",
```

Comment:
Missed these ones.

Reply:
Damn it, after I managed to get lucky with the damned snapshot.debian.org bug ... :) Thanks.